src/async_channel.rs

```diff
@@ -5,11 +5,22 @@ use futures::{
     Sink,
     Stream,
 };
+use log::*;
 use std::{pin::Pin, sync::Arc, task::Poll};
 
-pub fn bounded<T>(size: usize) -> (Publisher<T>, Subscriber<T>) {
-    let (sender, receiver) = raw_bounded(size);
+const LOG_TARGET: &str = "pub_sub";
```

Comment:
```suggestion
const LOG_TARGET: &str = "tari_broadcast_channel::async_channel";
```

LOG_TARGET should direct you to the source file